javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,74 @@
+//Future Age Calculator
+console.log("Sample program to find future age");
+let yearOfBirth=1987;
+let yearFuture=2027;
+let age = yearFuture-yearOfBirth;
+console.log(`You will be ${age} years old in ${yearFuture}`);
+
+// Dog Age Calculator
+let dogYearOfBirth=1975;
+let dogYearFuture=2045;
+let dogYear;
+ 
+function dogYearcalculator(shouldShowResultInDogYears)
+{
+if(shouldShowResultInDogYears===true)
+{
+dogYear=(dogYearFuture-dogYearOfBirth)/7;
+}
+else if(shouldShowResultInDogYears===false)
```

Comment:
since the value passed can only be true or false, you can write 
else {....}. No need for else if(shouldShowResultInDogYears===false)